yarn.lock

```diff
@@ -19302,9 +19302,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"react-dnd@patch:react-dnd@npm:16.0.1#./patches/react-dnd.patch::locator=lichtblick%40workspace%3A.":
+"react-dnd@patch:react-dnd@npm:16.0.1#./patches/react-dnd.patch::locator=foxbox%40workspace%3A.":
   version: 16.0.1
-  resolution: "react-dnd@patch:react-dnd@npm%3A16.0.1#./patches/react-dnd.patch::version=16.0.1&hash=7ebec9&locator=lichtblick%40workspace%3A."
+  resolution: "react-dnd@patch:react-dnd@npm%3A16.0.1#./patches/react-dnd.patch::version=16.0.1&hash=7ebec9&locator=foxbox%40workspace%3A."
```

Comment:
Please make sure to run yarn install after changing package.json, so these changes will not be present in the PR diff

Reply:
done